### PR TITLE
Linter testcase

### PR DESCRIPTION
**Pull request by UmerZia**
Apply the linter test case on "Hello Microverse" project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
...
+# Hello Microverse
+
+> Apply linter testcase on HTML/CSS.
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters: Stylelint, WebHint, LightHouse
```
Comment (Ugboaja-Uchechi) [inline README.md]: - [ ] Please make sure you add all necessary sections to your `README` file suggested by [Microverse](https://github.com/microverseinc/readme-template)🙏 .  Like the 

> **Author**
> **Contributing**
> **Show your support**
> **Acknowledgement and** 
> **License**

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#Excludes the test file from the repository
+test.md
+#Exclude node modules
```
Comment (Ugboaja-Uchechi) [inline .gitignore:3]: - [ ] Please make sure you add a `test.md` file locally and it is being ignored by GitHub 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @UmerZia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- Your repo looks professional ✔️ 
- Nice commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (UmerZia) [reply README.md]: I have changed the readme file accordingly.

File: .gitignore (hunk 2 above)
Comment (UmerZia) [reply .gitignore:3]: I added that file into my commit and it is ignored by GitHub.

**Comment by UmerZia** [comment]
@Ugboaja-Uchechi I have changed my project according to the requirements.

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @UmerZia 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**